From c83a22eb060277aedb000bbe5676f84770175499 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Wed, 25 Mar 2015 10:24:01 +0800
Subject: [PATCH] MLK-10464-4 ARM: dts: imx7d.dtsi: change the ADC compatible

The ADC in i.MX7D soc don't compatible with vf610_adc.

If not change this, for i.MX7D, during kernel boot up, the ADC
device of i.MX7D will match with vf610_adc driver, and make
the ADC device can't work.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index b3c6b111..49e870d 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -594,7 +594,7 @@
 			ranges;
 
 			adc1: adc@30610000 {
-				compatible = "fsl,imx7d-adc", "fsl,vf610-adc";
+				compatible = "fsl,imx7d-adc";
 				reg = <0x30610000 0x10000>;
 				interrupts = <GIC_SPI 98 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX7D_CLK_DUMMY>;
@@ -604,7 +604,7 @@
 			};
 
 			adc2: adc@30620000 {
-				compatible = "fsl,imx7d-adc", "fsl,vf610-adc";
+				compatible = "fsl,imx7d-adc";
 				reg = <0x30620000 0x10000>;
 				interrupts = <GIC_SPI 99 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX7D_CLK_DUMMY>;
-- 
1.8.0

